backend/spec/features/admin/configuration/countries_spec.rb

```diff
@@ -9,7 +9,7 @@ module Spree
       click_link "New Country"
 
       fill_in "Name", with: "Brazil"
-      fill_in "Iso Name", with: "BRL"
+      fill_in "ISO Name", with: "BRL"
```

Comment:
This prompted an in-office conversation over here, with @jarednorman chiming in something to the effect of: 

> I dislike computed expected values, which is what I consider that to be

Which I generally agree with. I'd say we leave it with the explicit translation and change the tests as we change the locales rather than testing on the computed value of Spree::Something.human_attribute_name. I don't expect the tests to break often, nor be hard to fix.